src/main/java/com/redhat/devtools/intellij/tektoncd/tree/ResourcesNode.java

```diff
@@ -26,7 +26,7 @@ public void load() {
         super.load();
         try {
             NamespaceNode namespaceNode = (NamespaceNode) getParent();
-            ((TektonRootNode)getRoot()).getTkn().getResources(namespaceNode.toString()).forEach(resource -> add(new ResourceNode(resource)));
+            ((TektonRootNode)getRoot()).getTkn().getResources(namespaceNode.toString()).forEach(resource -> add(new ResourceNode(resource.name())));
```

Comment:
I don't know the rationale for only keeping the name of the resource but I think it would be good to store the resource as we may want in the future to display other attributes (ex an icon based on the type of resource)